packages/wrangler/src/docs/index.ts

```diff
@@ -1,55 +1,49 @@
-import { printWranglerBanner } from "..";
-import { readConfig } from "../config";
+import { defineCommand } from "../core";
 import { logger } from "../logger";
-import * as metrics from "../metrics";
 import openInBrowser from "../open-in-browser";
 import { runSearch } from "./helpers";
-import type {
-	CommonYargsArgv,
-	StrictYargsOptionsToInterface,
-} from "../yargs-types";
 
-export function docsOptions(yargs: CommonYargsArgv) {
-	return yargs
-		.positional("search", {
+defineCommand({
+	command: "wrangler docs",
+	metadata: {
+		description: "📚 Open Wrangler's command documentation in your browser\n",
+		owner: "Workers: Authoring and Testing",
+		status: "stable",
+	},
+	args: {
+		search: {
 			describe:
 				"Enter search terms (e.g. the wrangler command) you want to know more about",
 			type: "string",
 			array: true,
-		})
-		.option("yes", {
+		},
+		yes: {
 			alias: "y",
 			type: "boolean",
-			description: "Takes you to the docs, even if search fails",
-		});
-}
+			describe: "Takes you to the docs, even if search fails",
+		},
+	},
+	positionalArgs: ["search"],
+	async handler(args) {
+		//if no command is provided, open the docs homepage
+		//or, if a command IS provided, but we can't find anything, open the docs homepage
+		let urlToOpen =
+			args.yes || !args.search || args.search.length === 0
+				? "https://developers.cloudflare.com/workers/wrangler/commands/"
+				: "";
 
-export async function docsHandler(
-	args: StrictYargsOptionsToInterface<typeof docsOptions>
-) {
-	//if no command is provided, open the docs homepage
-	//or, if a command IS provided, but we can't find anything, open the docs homepage
-	let urlToOpen =
-		args.yes || !args.search || args.search.length === 0
-			? "https://developers.cloudflare.com/workers/wrangler/commands/"
-			: "";
+		if (args.search && args.search.length > 0) {
+			const searchTerm = args.search.join(" ");
+			const searchResult = await runSearch(searchTerm);
 
-	if (args.search && args.search.length > 0) {
-		const searchTerm = args.search.join(" ");
-		const searchResult = await runSearch(searchTerm);
+			urlToOpen = searchResult ?? urlToOpen;
+		}
 
-		urlToOpen = searchResult ?? urlToOpen;
-	}
+		if (!urlToOpen) {
+			return;
+		}
 
-	if (!urlToOpen) {
-		return;
-	}
-	await printWranglerBanner();
-
-	logger.log(`Opening a link in your default browser: ${urlToOpen}`);
-	await openInBrowser(urlToOpen);
-	const config = readConfig(undefined, args);
-	await metrics.sendMetricsEvent("view docs", {
-		sendMetrics: config.send_metrics,
-	});
```

Comment:
removing this in preparation for new telementry

Reply:
Would it make sense to keep this until we refactor the telemetry code? It might be easier to justify removing it later, and to give us flexibility in case we decide to keep it last minute.

Reply:
yeah makes sense